server/src/main/java/org/elasticsearch/common/logging/ESJsonLayout.java

```diff
@@ -66,23 +66,25 @@
  * Once an appender is defined to be overriding a field, all the log events should contain this field.
  * <p>
  * The value taken from ESLogMessage has to be a simple escaped JSON value.
+ * @deprecated ECSJsonlayout should be used as JSON logs layout
  */
+@Deprecated(since = "v8")
 @Plugin(name = "ESJsonLayout", category = Node.CATEGORY, elementType = Layout.ELEMENT_TYPE, printObject = true)
 public class ESJsonLayout extends AbstractStringLayout {
 
     private final PatternLayout patternLayout;
-    private String overridenFields;
+    private String esmessagefields;
```

Comment:
Camel-case, perhaps?

```suggestion
    private String esMessageFields;
```

Reply:
I made this lowercase because a key in a property file is lowercase. Do you think it makes sense to keep it lowercase only because of this? Or camel-case is better?